debugger/main.ml

```diff
@@ -229,7 +229,14 @@ let main () =
     if !Parameters.version
     then printf "\tOCaml Debugger version %s@.@." Config.version;
     Loadprinter.init();
-    Load_path.init !default_load_path;
+    let secondary_dirs =
+      if !Clflags.no_std_include then []
+      else
+        let expand = Misc.expand_directory Config.standard_library in
+        Location.initialise_auto_include_alerts expand
+          Config.automatically_included_otherlibs
+    in
+    Load_path.init ~secondary_dirs !default_load_path;
```

Comment:
I wonder if it wouldn't be simpler overall to pass into `Load_path.init` a function that can report these new alerts. This way, we can be sure that the alert is always reported when needed, and the code wouldn't be split between load_path.ml, location.ml, and code sprinkled all over.

Reply:
Yes, that's a very nice way of short-circuiting it, thanks!